docs/data/base/components/autocomplete/autocomplete.md

```diff
@@ -0,0 +1,179 @@
+---
+product: base
+title: React Autocomplete hook
+hooks: useAutocomplete
+githubLabel: 'component: autocomplete'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/
+---
+
+# Autocomplete
+
+<p class="description">The Autocomplete component is a text input enhanced by a panel of suggested options.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+{{"component": "modules/components/ComponentPageTabs.js"}}
+
+## Introduction
+
+The autocomplete component is an enhanced text input that shows a list of suggested options as users type and lets them select an option from the list. It implements the WAI-ARIA Combobox pattern, and is typically used to assist users complete a form inputs or search queries faster.
+
+{{"demo": "AutocompleteIntroduction.js", "defaultCodeOpen": false, "bg": "gradient"}}
+
+:::warning
+Material UI and Joy UI have Autocomplete components that are built using the useAutocomplete hook, with many features that are not demo-ed here yet.
+
+To learn more about implementing a custom Autocomplete, you can explore the useAutocomplete API [docs](/base/react-autocomplete/hooks-api/), or reference the Material UI and Joy UI implementations:
+
+- [Material UI Autocomplete](/material-ui/react-autocomplete/)
+- [Joy UI Autocomplete](/joy-ui/react-autocomplete/)
+
+:::
```

Comment:
@samuelsycamore I think the message in this callout is a little clumsy, any suggestions? 😅

Reply:
Looks pretty good to me! Here's how I'd phrase that first part.

```suggestion
:::warning
Material UI and Joy UI have Autocomplete components that are built using the `useAutocomplete` hook, and they include many features not yet described here.

To learn more about implementing a custom Autocomplete, you can explore the [`useAutocomplete` API docs](/base/react-autocomplete/hooks-api/), or reference the Material UI and Joy UI implementations:

- [Material UI Autocomplete](/material-ui/react-autocomplete/)
- [Joy UI Autocomplete](/joy-ui/react-autocomplete/)

:::
```